### DISCUSSION TITLE
运行最新antdpro项目，安装moment-timezone插件后一直报Cannot read properties of undefined (reading 'split')是什么原因

### DISCUSSION THREAD

**Discussion by 18319700115** [umijs/umi#11945] [Q&A]
运行最新antdpro项目，安装moment-timezone插件后一直报Cannot read properties of undefined (reading 'split')是什么原因，

app.tsx新增这两行代码
import moment from 'moment-timezone';
console.info("moment=>>", moment)

**Comment by fz6m** [comment]
`moment-timezone` 只是 moment 的一个插件，需要同时具有 moment 依赖。

如有问题，需要给一个最小复现。
